source/excerptexport/views.py

```diff
@@ -178,7 +178,7 @@ def get_export_options(requestPostValues, optionConfig):
     return export_options
 
 
-@login_required(login_url='/excerptexport/login/')
+@login_required()
```

Comment:
it would be enough to wirte `@login_required`, no parentheses - but I like explicit better, so unless this is bothering in any way, I'd leave it as is.